index.html

```diff
@@ -10,25 +10,148 @@
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <div class="box1">
+        <img src="/image/Snapshoot Portfolio (3).png">
+        <h1>Multi-Post<br>Stories
+        <ul>
+            <li class="point1">CANOPY</li>
+            <li class="point2">Back End Dev</li>
+            <li class="point3">2015</li>
+        </ul>
+    </h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <!-- About me Section -->
+    <div class="about">
+        <div class="base">
+            <div class="top">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social">
+            <a href="#">LET'S CONNECT</a>
+            <ul class="icons">
                 <li><img src="/image/Vector.svg"></li>
                 <li><img src="/image/Linkedin icon.svg"></li>
                 <li><img src="/image/Vector (1).svg"></li>
                 <li><img src="/image/Vector (2).svg"></li>
                 <li><img src="/image/Vector (3).svg"></li>
-
             </ul>
-    </section>
-</body>
+        </div>
+        <div class="btn">
+            <a href="#" class="resume">Get my resume</a>
+        </div>
+        <div class="skill_list">
+            <div class="title1">
+                <h1>Languages</h1>
+                <a href="#" class= "fa fa-angle-down"></a>
+                <div class="langlist">
+                <ul class="list1">
+                    <img src="/image/Ellipse 1.png">
+                    <p>JavaScript</p>
+                </ul>
+                <ul class="list2">
+                    <img src="/image/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </ul>
+                <ul class="list3">
+                    <img src="/image/Ellipse 1 (2).png">
+                    <p>CSS</p>
+                </ul>
+                </div>
+            </div>
+            <div class="title2">
+                <h1>Frameworks</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+            <div class="title3">
+                <h1>Skills</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+        </div>
+        </div>
+    </div>
```

Comment:
- [ ] Nice work so far 👍🏿 as suggested by the previous reviewer Kindly use place your Language, Framework, and Skills in an unordered list, as h1 is not applicable in this case - <ul><li></li></ul>. Semantically, all similar items should be grouped together so that your styles will be very easy to apply in order to avoid repeating CSS values and properties.
 
 
 - [ ] Your design 
 
 
![image](https://github.com/Nomarg/My-Portfolio/assets/68823331/446164e8-d2b1-432f-8058-27f59613a478)

 - [ ] Figma Design
 
 
![image](https://github.com/Nomarg/My-Portfolio/assets/68823331/e35c7ce2-5a24-4e21-a4cc-4c043533ec25)